docs/README.md

```diff
@@ -0,0 +1,62 @@
+# Open Source Programs Web Docs
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+## Installation
+
+```console
+yarn install
+```
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+
+## Build
+
+```console
+yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+## Deployment
+
+```console
+GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+## Deploying to Surge
+
+Surge is a static web hosting platform. Tt is used to deploy our spen source programs docusaurus from the command line in a minute.
+
+Deploy using surge with the following steps:
+
+1. Install Surge using npm by running the following command:
+    ```
+    npm install --g surge
+    ```
+2. To build the static files of the site for production in the root/docs directory of project, run:
+    ```
+    npm run build
+    ```
+3. Run this command inside the root/docs directory of project:
+    ```
+    surge build/
+    ```
+    or 
+    You can deploy the site to existing domain using the command:
```

Comment:
In this or & You can deploy do not appear on different lines in the preview
![doc1](https://user-images.githubusercontent.com/45794450/108521269-8e98de80-72f1-11eb-9df9-d867ba5783df.JPG)
you could just mention Or, you can deploy . . .